addons/account/models/res_config_settings.py

```diff
@@ -43,7 +43,7 @@ class ResConfigSettings(models.TransientModel):
         string='Bank Suspense Account',
         readonly=False,
         related='company_id.account_journal_suspense_account_id',
-        domain=lambda self: "[('deprecated', '=', False), ('company_id', '=', company_id), ('user_type_id.type', 'not in', ('receivable', 'payable')), ('user_type_id', '=', %s)]" % self.env.ref('account.data_account_type_current_liabilities').id,
+        domain=lambda self: "[('deprecated', '=', False), ('company_id', '=', company_id), ('user_type_id.type', 'not in', ('receivable', 'payable')), ('user_type_id', 'in', %s)]" % [self.env.ref('account.data_account_type_current_assets').id, self.env.ref('account.data_account_type_current_liabilities').id],
```

Comment:
(tsb) told me to keep them both so that the customers already using `data_account_type_current_liabilities` don't send tickets to us, but if it has no effect I can remove it, what do you think ?

Reply:
@vin-odoo can you please r+ if everything is ok ?

Reply:
Hi I can't r+ :smile: But I've forwarded the PR to the team they will most likely take another look today and r+ if it's ok with them too.